util/metrics/prometheus.py

```diff
@@ -65,6 +65,10 @@
     "seconds taken to make an index request to the secscan service",
 )
 
+# secscan latency
+secscan_result_duration = Histogram(
+    "quay_secscan_result_duration", "seconds taken to retrieve secscan results from service"
```

Comment:
You're currently measuring the time between a manifest gets created and the time the security worker sends it for indexing.  So you're not really getting the time the service takes to actually create the report (which I think we can get from the current Clair metrics available). Wondering if having the number of outstanding manifests to be indexed by the securityworker would be a better measure?

Reply:
Or even the throughput of the securityworker?

Reply:
Ah ok, @syed thoughts on this? I think the original goal was to have a way to track how long it takes to see vulnerabilities reported after pushing an image. Maybe that should be time from manifest to security worker + time to create a report (from Clair metrics)?